AppCenter/AppCenterTests/MSDeviceTrackerTests.m

```diff
@@ -268,6 +268,34 @@ - (void)testNoCarrierCountry {
 }
 #endif
 
+#if TARGET_OS_IOS
+- (void)testCarrierCountryNotOverridden {
+  // If
+  NSString *expected = @"US";2
```

Comment:
Delete `2` at end of line

Reply:
Whoops! I've just started using a new mechanical keyboard and it's wreaking havoc with typos. Fixed now 😁